spec/controllers/api/welcome_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe WelcomeController, type: :controller do
+RSpec.describe Api::WelcomeController, type: :controller do
```

Comment:
what is the reason to do this?

Reply:
no reason to do it, i tried to fix some bugs before and forgot remove it